usl_models/tests/atmo_ml/atmo_model_test.py

```diff
@@ -0,0 +1,73 @@
+"""Tests for Atmo model."""
+
+import tensorflow as tf
+
+from usl_models.atmo_ml import model as atmo_model
+from usl_models.atmo_ml import constants
+from usl_models.atmo_ml import model_params
+
+_TEST_MAP_HEIGHT = 200
+_TEST_MAP_WIDTH = 200
+_TEST_SPATIAL_FEATURES = 40
+_TEST_SPATIOTEMPORAL_FEATURES = 6
+
+
+def fake_input_batch(
+    batch_size: int,
+    height: int = _TEST_MAP_HEIGHT,
+    width: int = _TEST_MAP_WIDTH,
+) -> dict[str, tf.Tensor]:
+    """Creates a fake training batch for testing.
+
+    Args:
+        batch_size: Batch size.
+        height: Optional height.
+        width: Optional width.
+
+    Returns:
+        A dictionary of model inputs, with the following key/value pairs:
+        - "spatial": Required. A rank-4 tensor [batch, height, width, features].
```

Comment:
Sure, I can define a TypedDict for this function. But I'm not sure it would be useful for the model in general, depending on how the data generation will look, so this might create some inconsistency?

Reply:
Inconsistent in what way?
You'd still be returning a dictionary just as you are currently, the TypedDict would just supply some type hints and provide the documentation on what fields will be present in the dictionary you're currently placing in the docstrings.

Reply:
Ah sorry, was thinking about dataclasses. Updated.